app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="post">
+  <%= link_to user_post_path(@user, post), id: "user_#{post.id}", class: 'posts-container' do %> 
+    <div class="post-header">
+      <h3>Post <%= post.id %></h3>
+    </div>
+    <div class="post-details">
+      <p><%= truncate(post.text, length: 100) %></p>
+    </div>
+    <div class="comments-likes">
+      <p>Comments: <%= post.comments.count %></p>
+      <p>Likes: <%= post.likes.count %></p>
+    </div>
+  <% end %>
+</div>
+<ul class="comments-container">
+  <% if post.comments.present? %>
+    <% post.comments.each do |comment| %> 
```

Comment:
- [ ] I see this is a partial used in the post index page. Kindly note that the comments that should be listed on the post index page are the `recent comments`, not all the comments. Kindly fix that.

Reply:
Fixed. Thanks